mappings/84c1b4ee5762b0407b07f3b9127f8744ad51b69961ce317f6d22ff7f.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "84c1b4ee5762b0407b07f3b9127f8744ad51b69961ce317f6d22ff7f",
+    "name": {
+        "sequenceNumber": 0,
+        "value": "DBOB",
+        "signatures": [
+            {
+                "signature": "35395d316aa5194bde508238e32750ae359a73ec88c90971db5523f56248da5221212a18e9c4f7a5be6e67cccf13162b22d0c8c3ef711edc1e1d60d3b999eb00",
+                "publicKey": "39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f"
+            }
+        ]
+    },
+    "ticker": {
+        "sequenceNumber": 0,
+        "value": "DBOB",
+        "signatures": [
+            {
+                "signature": "7a38a7ff3af8b6eba3219c326de93f2eac203100a0cbdba5af7be6d4b78f65deae35cb0d3e80ccd3b925deb341ebc67febef89e20c3eda482ea8bfb5834b6c05",
+                "publicKey": "39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f"
+            }
+        ]
+    },
+    "decimals": {
+        "sequenceNumber": 0,
+        "value": 0,
+        "signatures": [
+            {
+                "signature": "715398ca2074a6fe73dfb10300bdc4b96ea585d1afbf3fcc02fba5864879d3206a3430964ec9d02da08bb7acb38f0ca8c191fd6ce49cc95ee98ed14dcd536900",
+                "publicKey": "39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f"
+            }
+        ]
+    },
+    "policy": "84c1b4ee5762b0407b07f3b9127f8744ad51b69961ce317f6d22ff7f",

```

Comment:
Hi @xcoder92,

validator can't proceed because you are putting policyID in the policy field, but it should be a base16-encoded CBOR representation of the monetary policy script. 

Please, update the PR.